experimental/tf_proxy/client/build.rs

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use oak_utils::{generate_grpc_code, CodegenOptions};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    generate_grpc_code(
+        "../../../third_party/google",
+        &["tensorflow_serving/apis/predict.proto"],
```

Comment:
```suggestion
        "../../../",
        &["third_party/google/tensorflow_serving/apis/predict.proto"],
```

To be consistent with the rest of the code and follow the recommendation that paths should be specified relative to the root of the repo.